eth/client.go

```diff
@@ -602,8 +593,8 @@ func (c *client) Vote(pollAddr ethcommon.Address, choiceID *big.Int) (*types.Tra
 		return nil, err
 	}
 
-	gl, gp := c.GetGasInfo()
-	opts, err := c.accountManager.CreateTransactOpts(gl, gp)
+	// Setting gasLimit and gasPrice to zero, nil allows it to be overwritten by eth.Backend
```

Comment:
Why not just leave the existing code as-is since gasLimit will default to 0 and gasPrice will default to nil anyway especially since all the other contract methods will continue to use the stored gasLimit and gasPrice?